### PR TITLE
Changed formatting of WPF article

### PR DESCRIPTION
**Pull request by rpetrusha**
## Changed formatting of WPF article

In reviewing #13573, I noticed that level3 heads were used, making them children of the level2 Prerequisites section. This PR changes that. It:

- Elevates level3 heads to level2.
- Removes annoying infinitives in the header text.
- Fully qualifies a method name to remove awkward *typename* method *methodname*.
- Replaces a FWLink.

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md [hunk 1]
```diff
@@ -15,35 +15,35 @@ If you don't want to build the applications yourself, you can download "Async Sa
...
 
-- [To replace method GetURLContentsAsync with a .NET Framework method](#GetURLContentsAsync)
+- [Replace the GetURLContentsAsync method with a .NET Framework method](#GetURLContentsAsync)
 
 - [Example](#BKMK_CompleteCodeExamples)
 
 ## Prerequisites
 
-Visual Studio 2012 or later must be installed on your computer. For more information, see the [Microsoft website](https://go.microsoft.com/fwlink/?LinkId=235233).
+Visual Studio 2012 or later must be installed on your computer. For more information, see the Visual Studio [Downloads](https://visualstudio.microsoft.com/downloads/) page.
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md]: we should use campaign links when linking to VS downloads - I might add this info to the contributor guide as well

```suggestion
Visual Studio 2012 or later must be installed on your computer. For more information, see the Visual Studio [Downloads](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) page.
```

File: docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md [hunk 2]
```diff
@@ -15,35 +15,35 @@ If you don't want to build the applications yourself, you can download "Async Sa
 
 In this walkthrough, you complete the following tasks:
 
-- [To create a WPF application](#CreateWPFApp)
+- [Create a WPF application](#CreateWPFApp)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md]: Perhaps add this to a checklist like in the mvc style?
https://github.com/dotnet/docs/blame/master/docs/machine-learning/tutorials/movie-recommendation.md#L17
https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommendation

Then we could also get rid of the custom anchors.

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments for you to consider.

File: docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md (hunk 2 above)
Comment (rpetrusha) [reply docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md]: Thanks, @Maira. The checklist looks great, and I've gotten rid of all the custom anchors.

**Pull request merged**